### ISSUE TITLE
[a11y]: Tooltip is with violation "Content is not within a landmark element"

### ISSUE THREAD

**Issue by chubbywhale** [carbon-design-system/carbon#12047]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

7.59.0

### React version

18.3.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

The IBM Accessibility Checker reports that the `span` that contains the Focus sentinel content that is generated by the Tooltip component violated the rule `Content is not within a landmark element `

<img width="365" alt="image" src="https://user-images.githubusercontent.com/37340587/187820808-07092ae1-d8df-4fb7-999f-45bddfe731e2.png">

### Reproduction/example

https://1fsxyd.csb.app/

### Steps to reproduce

Execute the IBM Accessibility Checker scan when the Toolitp is open, then the scan should report the above violation.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
There was a discussion about something similar [in slack recently](https://ibm-studios.slack.com/archives/C0M053VPT/p1663853855563719?thread_ts=1662595315.560349&cid=C0M053VPT).

Generally these are considered false positives, but it sounds like we might have this issue because the focus sentinels are rendered as siblings to the element given a `role="alertdialog"`. In a similar case of Modal, the focus sentinels are rendered as siblings of the element given `role="dialog"`.

So generally speaking this is a false report from accessibility checker that can probably be ignored. That said, we could explore here being more precise in placing the focus sentinels within the elements given a role of `alertdialog` or `dialog`.

**Comment by tay1orjones** [comment]
For now we're going to classify this as a false positive after discussing with IBMa
